### ISSUE TITLE
[FEATURE REQUEST] Remove "master" from all contexts (Git branches, config files, and code comments)

### ISSUE THREAD

**Issue by msulprizio** [geoschem/geos-chem#376]
@ltmurray wrote:

>As some of you may have seen [in the news](https://www.bbc.com/news/technology-53050955), in response to the recent and ongoing Black Lives Matter protests in the United States and abroad, Github is now promoting changing the default branch name for repositories from “master” to something with more neutral connotations like “main” or “root”.
>
>Given the fact that the Geosciences have the worst diversity statistics of any STEM field (e.g., doi:10.1038/s41561-018-0116-6), I was wondering if the GCSC would be willing to support renaming our master branch to a more neutral term.  (And as we move more into MPI with GCHP, use terms like “main" and “dependent", or “root” and “branch” in lieu of “master” and “slave”).  
>
>My opinion is that this is a simple semantic change that has no impact on our science or coding, but removes a potential microagression that otherwise could contribute to the leaky pipe of underrepresented scientists in our field.
>
>Thank you for your consideration.

**Comment by yantosca** [comment]
We will also update our documentation accordingly for the 13.0.0 release.

**Comment by lizziel** [comment]
The main branch is now available for use, synced to master. The master branch will be deleted in 13.0.

**Comment by ltmurray** [comment]
Thanks, Lizzie!!

**Comment by lizziel** [comment]
Of course! I also want to link to this [blog post](https://www.hanselman.com/blog/EasilyRenameYourGitDefaultBranchFromMasterToMain.aspx) as a resource for how to rename master to main in your own projects. Since GEOS-Chem is actively being used right now I simply created a new branch off of master called main and pushed it, with subsequent steps to delete coming later. However, this post goes through how to change the name in one step. The information about handling your existing clone is also helpful. GCST will distribute our own information with the same steps for the release of GEOS-Chem 13.0.0.

**Comment by yantosca** [comment]
You can use a bash script (such as the sample below) to rename `master` to `main` in your own Github projects.  This builds off the [blog post](https://www.hanselman.com/blog/EasilyRenameYourGitDefaultBranchFromMasterToMain.aspx) mentioned by @lizziel above.

```
#!/bin/bash

### Script to rename "master" Git branch to "main".

# Move master to main
git branch -m master main

# Push main to remote repo
git push -u origin main

# Remove reference to remote master branch
git branch -r -d origin/master

# Echo info
echo "You also need to update the default branch at the remote Github repo!!!"
echo "Select 'branches', then 'Change default branch', and then pick 'main'."
```

**Comment by yantosca** [comment]
We have now removed the terms "master" and "slave" from all GEOS-Chem source code files and run directory files.  This was resolved in commits 7f8b5a3b8fd2e0bd583e620448b2b05f6df8c60d and 4c2ff56cd1b4883b9dc3696838494a4f147d139b.

We still will keep "master" in GEOS-Chem until the 13.0.0 release, at which time "main" will become the default branch.  This is necessary for backwards compatibility (e.g. for pull requests made with older versions, etc.)
